src/main/resources/assets/pyrotech/lang/zh_cn.lang

```diff
@@ -350,16 +352,18 @@ gui.pyrotech.tooltip.torch.burns.false=持续燃烧。
 gui.pyrotech.tooltip.rain.extinguishes.true=淋雨后熄灭，
 gui.pyrotech.tooltip.rain.extinguishes.false=淋雨后仍燃烧。
 gui.pyrotech.tooltip.potency=Potency: %d%%
```

Comment:
The potency tooltip describes the magnitude of the effect that eating a Roasted Marshmallow has.